modules/indigo/module/inc/indigo/port_manager.h

```diff
@@ -225,6 +225,22 @@ extern indigo_error_t indigo_port_queue_config_get(
     of_queue_get_config_reply_t **queue_config_reply);
 
 
+/**
+ * @brief Process an OF queue stats request handler
```

Comment:
Maybe reword this to: "Prototype of an OF queue stats request handler; if defined, this handler will be called when a queue stats request is received"